### PR TITLE
add cli installation instructions for macos and windows

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: sources/tools/cli/index.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ The Reliably CLI is a command-line tool for working with Reliably. The Reliably
 
 If you'd like to use Reliably locally then you can install and use the Reliably CLI. You can grab the right executable for your operating system from [Reliably CLI Releases][releases].
 
+!!! note
+    For detailed installtion instructions for different operating systems please see
```
Comment (mperrien) [inline sources/tools/cli/index.md:13]:
```suggestion
    For detailed installation instructions for different operating systems please see
```

File: sources/tools/cli/index.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ The Reliably CLI is a command-line tool for working with Reliably. The Reliably
...
+    [Installation Details](#instalation-details).
```
Comment (mperrien) [inline sources/tools/cli/index.md:14]:
```suggestion
    [Installation Details](#installation-details).
```

File: sources/tools/cli/index.md [hunk 3]
```diff
@@ -96,3 +100,108 @@ with the `--output` or `-o` flag, as follow:
...
+Get-FileHash .\reliably.exe -algorithm md5
+```
+You should see output similar to:
+
+![Md5 Screenshot](./images/md5-screenshot.png)
+
+Compare the md5 checksum displayed with the md5 checksum in the
+[MD5 checksum file](windows10-release-md5), they should match.
+
+To execute ```reliably CLI``` you will need to rename the file and add a ```.exe.``
```
Comment (mperrien) [inline sources/tools/cli/index.md:172]:
```suggestion
To execute ```reliably CLI``` you will need to rename the file and add a ```.exe.```
```

File: sources/tools/cli/index.md [hunk 4]
```diff
@@ -96,3 +100,108 @@ with the `--output` or `-o` flag, as follow:
...
+
+Then execute:
+
+```powershell
+.\relliably.exe --version
+```
+
+You should see the current ```reliably CLI```  version displayed.
+
+At this point to make reliably convenient to run add the folder to the
```
Comment (mperrien) [inline sources/tools/cli/index.md:187]:
```suggestion
At this point, to make Reliably convenient to run, add the folder to the
```

File: sources/tools/cli/index.md [hunk 5]
```diff
@@ -96,3 +100,108 @@ with the `--output` or `-o` flag, as follow:
...
+executable path. From the windows Icon right click and select ```System``` from
```
Comment (mperrien) [inline sources/tools/cli/index.md:188]:
```suggestion
executable path. From the Windows icon right-click and select ```System``` from
```

**Review by mperrien — COMMENTED** [review]
@gtfisher I made a few suggestions.

I'm wondering about renaming the "Installing Reliably on MacOS" section to "Installing Reliably on MacOs and Linux systems". I guess it's identical.

File: sources/tools/cli/index.md (hunk 5 above)
Comment (gtfisher) [reply sources/tools/cli/index.md:188]: I think "Windows start menu" is better in hindsight

File: sources/tools/cli/index.md [hunk 6]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+
+* Environment Variables
+* In the Enviroment Variables pop-up, select Path from the User environment variables
+* Select Edit then
+* Select Browse...
+* Navigate to the folder where you downloaded the reliably executable
+* Select OK
+
+Open a new Powershell window and type ```reliably --version``, the version of
+the Reliably CLI shoulld be disaplyed.
```
Comment (mperrien) [inline sources/tools/cli/index.md:200]:
```suggestion
the Reliably CLI should be displayed.
```

File: sources/tools/cli/index.md [hunk 7]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+move .\reliably-windows-amd64 .\relliably.exe
+```
+
+Then execute:
+
+```powershell
+.\relliably.exe --version
+```
+
+You should see the current ```reliably CLI```  version displayed.
```
Comment (mperrien) [inline sources/tools/cli/index.md:185]:
```suggestion
You should see the current ```reliably CLI``` version displayed.
```

File: sources/tools/cli/index.md [hunk 8]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+
+At this point to make Reliably convenient to run add the folder to the
```
Comment (mperrien) [inline sources/tools/cli/index.md:187]:
```suggestion
At this point, to make Reliably convenient to run, add the folder to the
```

File: sources/tools/cli/index.md [hunk 9]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+
+Open a terminal window and navigate to the directory where you downloaded the Reliably image.
+
+Check the md5 checksum to confirm the downloaded file has not been corrupted:
+
+```bash
+md5sum reliably-linux-amd64 
+
+cab791d7d4b4e5a0dcf00392b4ce672c  reliably-linux-amd64
+
```
Comment (mperrien) [inline sources/tools/cli/index.md:218]:
```suggestion
```

File: sources/tools/cli/index.md [hunk 10]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+```bash
+chmod u+x reliably-darwin-amd64
+```
+
+To confirm the file can be execute run:
+
+```bash
+./reliably-linux-amd64 --version
+
```
Comment (mperrien) [inline sources/tools/cli/index.md:232]:
```suggestion
```

File: sources/tools/cli/index.md [hunk 11]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+Open a new Powershell window and type ```reliably --version``, the version of
+the Reliably CLI shoulld be disaplyed.
+
+Reliably CLI will now be available in you Powershell sessions.
+
+
+### Installing Reliably on Linux (Ubuntu)
+
+
```
Comment (mperrien) [inline sources/tools/cli/index.md:207]:
```suggestion
```

File: sources/tools/cli/index.md [hunk 12]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+* Download  [Reliably Linux Image][reliably-linux] to a working directory.
```
Comment (mperrien) [inline sources/tools/cli/index.md:208]:
```suggestion
* Download [Reliably Linux Image][reliably-linux] to a working directory.
```

File: sources/tools/cli/index.md [hunk 13]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+```
+Compare the md5 checksum displayed with the [MD5 checksum file][reliably-linux-md5], they should match.
+
+Make the file and executable by executing:
+
+```bash
+chmod u+x reliably-darwin-amd64
+```
+
+To confirm the file can be execute run:
```
Comment (mperrien) [inline sources/tools/cli/index.md:228]:
```suggestion
To confirm the file can be executed, run:
```

File: sources/tools/cli/index.md [hunk 14]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+
+```bash
+./reliably-linux-amd64 --version
+
+```
+
+The Relibly CLI version should be disaplyed.
```
Comment (mperrien) [inline sources/tools/cli/index.md:235]:
```suggestion
The Reliably CLI version should be displayed.
```

File: sources/tools/cli/index.md [hunk 15]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+
+At this point to make Reliably convenient to run move the executable to a
```
Comment (mperrien) [inline sources/tools/cli/index.md:237]:
```suggestion
At this point, to make Reliably convenient to run, move the executable to a
```

File: sources/tools/cli/index.md [hunk 16]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+directory that is on your path for example ```/usr/local/bin``` and rename the
```
Comment (mperrien) [inline sources/tools/cli/index.md:238]:
```suggestion
directory that is on your path, for example ```/usr/local/bin```, and rename the
```

File: sources/tools/cli/index.md [hunk 17]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+file to ```reliably```.
+
+```bash
+sudo mv reliably-linux-amd64 /usr/local/bin/reliably 
+
```
Comment (mperrien) [inline sources/tools/cli/index.md:243]:
```suggestion
```

File: sources/tools/cli/index.md [hunk 18]
```diff
@@ -96,3 +100,161 @@ with the `--output` or `-o` flag, as follow:
...
+```
+
+Then open a new terminal window execute:
+
+```bash
+reliably --version
+```
+
+You should see the current ```reliably``` cli version disaplyed.
```
Comment (mperrien) [inline sources/tools/cli/index.md:252]:
```suggestion
You should see the current ```reliably``` cli version displayed.
```

**Review by mperrien — COMMENTED** [review]
I found a few typos, @gtfisher.
I made suggestions, hoping this makes accepting or refusing a single-click operation.

**Pull request merged**